Cargo.toml

```diff
@@ -72,18 +73,24 @@ termion = ["dep:termion"]
 termwiz = ["dep:termwiz"]
 
 #! The following optional features are available for all backends:
-## enables serialization and deserialization of style and color types using the [Serde crate].
-## This is useful if you want to save themes to a file.
+
+## enables serialization and deserialization of style, color and state types using the
+## [`serde`](https://crates.io/crates/serde) crate. This is useful if you want to save themes to
+## a file.
 serde = ["dep:serde", "bitflags/serde"]
 
 ## enables the [`border!`] macro.
 macros = []
 
-## enables all widgets.
-all-widgets = ["widget-calendar"]
+## Material design color scheme. Enables the [`material`](crate::style::material) module and adds a
+## dependency on the [`matdesign-color`](https://crates.io/crates/matdesign-color) crate.
+material = ["dep:matdesign-color"]
```

Comment:
I'm assuming we can include a feature for tailwindcss in the future?

Reply:
on the way...